### ISSUE TITLE
[Bug]: 1.9.3 vs 1.8.0 Inpainting Differences?

### ISSUE THREAD

**Issue by onewhoflies** [Bing-su/adetailer#602]
### Describe the bug

When using the exact same settings (seed, positive/negative prompt, sampler, etc.) on A111 1.8.0 and 1.9.3, inpainting a person using person_yolov8n-seg.pt in ADetailer 1st then inpainting a face using face_yolov8n.pt in ADetailer 2nd gives different results. I have already confirmed that 1.8.0 and 1.9.3 generate the same images with ADetailer disabled to try to isolate the issue.

### Steps to reproduce

(detailing what I did, everything may not be necessary to see the issue)
1. In txt2img, generate an image and use Hires Fix (2x, latent nearest-exact upscaler, denoising strength 0.55) in A111 1.8.0 with the DPM++ 3M SDE Karras sampler (on A111 1.8.0 - [https://github.com/AUTOMATIC1111/stable-diffusion-webui/releases/tag/v1.8.0](https://github.com/AUTOMATIC1111/stable-diffusion-webui/releases/tag/v1.8.0)), with ADetailer 1st and 2nd enabled using the person_yolov8n-seg and face_yolov8n.pt models, leaving the ADetailer 1st prompt empty and a basic prompt describing the face in ADetailer 2nd.
2. Repeat step 1, but use A111 1.9.3 - [https://github.com/AUTOMATIC1111/stable-diffusion-webui/releases/tag/v1.9.3](https://github.com/AUTOMATIC1111/stable-diffusion-webui/releases/tag/v1.9.3)

If any more info would be useful, I can provide it. Thanks for making such a useful extension.

### Console logs, from start to end.

```Shell
venv "I:\stable-diffusion-webui\venv\Scripts\Python.exe"
Python 3.10.11 (tags/v3.10.11:7d4cc5a, Apr  5 2023, 00:38:17) [MSC v.1929 64 bit (AMD64)]
Version: v1.8.0
Commit hash: bef51aed032c0aaa5cfd80445bc4cf0d85b408b5
Launching Web UI with arguments:
no module 'xformers'. Processing without...
no module 'xformers'. Processing without...
No module 'xformers'. Proceeding without it.
[-] ADetailer initialized. version: 24.4.2, num models: 20
ControlNet preprocessor location: I:\stable-diffusion-webui\extensions\sd-webui-controlnet\annotator\downloads
2024-05-01 22:31:46,791 - ControlNet - INFO - ControlNet v1.1.446
2024-05-01 22:31:46,844 - ControlNet - INFO - ControlNet v1.1.446
[sd-webui-freeu] Controlnet support: *enabled*
Loading weights [62bb78983a] from I:\stable-diffusion-webui\models\Stable-diffusion\epicphotogasm_lastUnicorn.safetensors
Creating model from config: I:\stable-diffusion-webui\configs\v1-inference.yaml
2024-05-01 22:31:47,762 - ControlNet - INFO - ControlNet UI callback registered.
Running on local URL:  http://127.0.0.1:7860

To create a public link, set `share=True` in `launch()`.
Startup time: 11.8s (prepare environment: 5.7s, import torch: 1.8s, import gradio: 0.6s, setup paths: 0.4s, initialize shared: 0.2s, other imports: 0.3s, load scripts: 1.7s, create ui: 1.0s, gradio launch: 0.1s).
Loading VAE weights specified in settings: I:\stable-diffusion-webui\models\VAE\realisticVisionV20_v20.safetensors
Applying attention optimization: sdp-no-mem... done.
Model loaded in 1.6s (load weights from disk: 0.2s, create model: 0.5s, apply weights to model: 0.7s).
100%|██████████████████████████████████████████████████████████████████████████████████| 30/30 [00:03<00:00,  9.35it/s]
100%|██████████████████████████████████████████████████████████████████████████████████| 10/10 [00:04<00:00,  2.12it/s]
postprocess_batch

0: 480x640 1 person, 67.0ms
Speed: 1.0ms preprocess, 67.0ms inference, 9.0ms postprocess per image at shape (1, 3, 480, 640)
100%|██████████████████████████████████████████████████████████████████████████████████| 13/13 [00:01<00:00,  7.61it/s]

0: 480x640 1 face, 4.0ms
Speed: 1.0ms preprocess, 4.0ms inference, 1.0ms postprocess per image at shape (1, 3, 480, 640)
100%|██████████████████████████████████████████████████████████████████████████████████| 14/14 [00:02<00:00,  6.34it/s]


(^ looks the same for 1.9.3)
```

**Comment by Bing-su** [comment]
Probably an undocumented [seed breaking change](https://github.com/AUTOMATIC1111/stable-diffusion-webui/wiki/Seed-breaking-changes), or maybe an effect of the sampler change in 1.9.0.

Adetailer doesn't do anything about it.
